### PR TITLE
MGDIRE2_SB-215

### PR DESCRIPTION
**Pull request by danduk82**
implements [MGDIRE2_SB-215](https://jira.swisstopo.ch/browse/MGDIRE2_SB-215)

### CODE REVIEW

**Review by ismailsunni — APPROVED** [review]
LGTM

File: kadasrouting/gui/valhallaroutebottombar.py [hunk 1]
```diff
@@ -310,9 +325,12 @@ def clearPins(self):
         """
         # remove origin pin
         self.originSearchBox.removePin()
-        # remove destination poin
+        # remove destination pin
         self.destinationSearchBox.removePin()
 
+    def deletePoints(self):
+        self.originSearchBox.point
```
Comment (volaya) [inline kadasrouting/gui/valhallaroutebottombar.py:332]: What is this supposed to do?

Comment (danduk82) [reply kadasrouting/gui/valhallaroutebottombar.py:332]: oups should be `self.originSearchBox.point = None` 
good catch, thanks

Comment (danduk82) [reply kadasrouting/gui/valhallaroutebottombar.py]: actually no, this was useless

**Pull request merged**